traefik/values.yaml

```diff
@@ -413,3 +413,18 @@ securityContext:
 
 podSecurityContext:
   fsGroup: 65532
+
```

Comment:
Seems that something went wrong and the below section has been duplicated.  Can you please try to use the following

```yaml
topologySpreadConstraints:
  - maxSkew: 1
    topologyKey: topology.kubernetes.io/zone
    whenUnsatisfiable: DoNotSchedule
    labelSelector:
      matchLabels:
        app.kubernetes.io/instance: traefik
```

Reply:
Weird, fixed